client/src/components/JobInformation/CodeRow.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+    <tr @click="toggleExpanded">
+        <td>
+            {{ codeLabel }}
+            <span v-if="codeItem && !expanded"><i>(Click to expand)</i></span>
```

Comment:
```suggestion
            <span v-if="codeItem && !expanded"></span>
```

Reply:
I guess we can remove the whole span, then right ?

Reply:
right :+1: